### PR TITLE
workaround unpredictable attached ordering

### PR DESCRIPTION
**Pull request by morethanreal**
Workaround until https://github.com/Polymer/polymer/issues/1773 is landed. For https://github.com/PolymerElements/paper-input/issues/53 PTAL @notwaldorf

### CODE REVIEW

File: paper-input-container.html [hunk 1]
```diff
@@ -369,6 +369,9 @@
     },
 
     ready: function() {
+      if (!this._addons) {
```
Comment (notwaldorf) [inline paper-input-container.html]: Is this needed? (won't `_onAddonAttached` do it on the first addon anyway?)

Comment (morethanreal) [inline paper-input-container.html]: Yeah, there could be no addons. I can lazily check for `this._addons` where it's referenced if you prefer.

Comment (notwaldorf) [inline paper-input-container.html]: Oh, I was thinking that if there's no addons, we don't care if it's an array, but we probably assume it is somewhere else in the code. It's fine like this :)

**Comment by notwaldorf** [comment]
lgtm

**Pull request merged**